ci/scripts/rust_example.sh

```diff
@@ -22,6 +22,15 @@ set -e
 export CARGO_PROFILE_CI_OPT_LEVEL="s"
 export CARGO_PROFILE_CI_STRIP=true
 
+# Remove leftover files inside test_* directories inside datafusion-examples/
```

Comment:
this is probably enough -- you don't need to also change the example to run in a temp dir as well,right?